docs/src/tutorials/sde_example.md

```diff
@@ -122,6 +122,9 @@ are added via `addprocs()`, but we can change this to use multithreading via
 sol = solve(ensembleprob,EnsembleThreads(),trajectories=1000)
 ```
 
+If you use a custom noise process, you might need to specify it in a custom `prob_func`
+in the `EnsembleProblem` constructor, as each trajectory needs its own noise process.
+
```

Comment:
```suggestion
```

!!! warn
    If you use a custom noise process, you might need to specify it in a custom `prob_func`
    in the `EnsembleProblem` constructor, as each trajectory needs its own noise process.

```